drivers/Kconfig

```diff
@@ -33,4 +33,8 @@ menu "Storage Device Drivers"
 rsource "mtd_sdcard/Kconfig"
 endmenu # Storage Device Drivers
 
+menu "Miscellaneous Device Drivers"
```

Comment:
Do you mind moving this before "Network Device Drivers"?

Reply:
Since it was Miscellaneous, I had it at the bottom.  Shall update